deps/GraphBLAS/Config/GraphBLAS.h.in

```diff
@@ -5557,8 +5557,8 @@ typedef struct
     GrB_Index nvals ;       // Number of none zero values in matrix
     GrB_Index nnz_idx ;     // Index of current none zero value
     int64_t p ;             // Number of none zero values in current column
-    int64_t col_idx ;       // Index of current column
-    GrB_Index ncols ;       // Total number of coulmns in matrix
+    int64_t idx ;           // Index of current row/col
+    GrB_Index n ;           // Total number of rows/col in matrix
```

Comment:
GrB_Index nrows ;       // Total number of rows in matrix

Reply:
done